Modules/Filtering/ImageStatistics/include/itkLabelOverlapMeasuresImageFilter.hxx

```diff
@@ -382,8 +382,12 @@ LabelOverlapMeasuresImageFilter<TLabelImage>::GetFalsePositiveError() const -> R
     {
       continue;
     }
-    numerator += static_cast<RealType>((*mapIt).second.m_SourceComplement);
-    denominator += static_cast<RealType>((*mapIt).second.m_Source);
+    LabelImagePointer sourceImg = const_cast<TLabelImage *>((*mapIt)->GetSourceImage());
+    auto nVox = sourceImg->GetBufferedRegion().GetNumberOfPixels(); //TP+FP+FN+TN
```

Comment:
Lines 385-386 are duplicated on 421-422. They should be moved up to line 378 (before the for loop).